.npmrc

```diff
@@ -1 +1 @@
-enable-pre-post-scripts=true
+enable-pre-post-scripts=true
```

Comment:
why remove new line here?  🥲

Reply:
I tried something then prettier kept that :/